### PR TITLE
ui: Service Mesh - Topology tab and basic layout

### PR DESCRIPTION
**Pull request by kaxcode**
This is the foundational layout of the Topology Tab with Service Mesh visualization.

🎉  Highlights
- TopologyMetrics component holds all the logic for making DOM calculations and dynamically drawing the SVG elements in the Topology tab.
- Integration of new endpoint `/v1/internal/ui/service-topology/`
- Helper that returns health checks in a percentage format
- Styling files live in component folder

🐛 Present bugs
✅ ~~To be fixed before merge. Ran into an ember issue when covering the use case, Topology Tab should not be displayed if Connect is not enabled.~~

To-do
- [x]  Merge and pull #8798 

topology

### CODE REVIEW

File: ui-v2/app/routes/dc/services/show/topology.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    const parent = this.routeName
+      .split('.')
+      .slice(0, -1)
+      .join('.');
+    return this.modelFor(parent);
+  },
+  setupController: function(controller, model) {
+    controller.setProperties(model);
+  },
+});
```
Comment (johncowen) [inline ui-v2/app/routes/dc/services/show/topology.js:14]: I know you did this before the Outlet loading PR, but now that is merged could you update this Route to use our base Route and add a `this._super` for setupController? 🙇‍♂️

**Review by johncowen — APPROVED** [review]
This is looking amazing! 👏  I left mainly a few non-important suggestions, and I know this is going to be iterated on. I think the most important thing is making sure the viz redraws when a blocking query comes in.

I'm gonna approve this as it looks great and is useful for users as is, and I know you'll be building on this moving forwards. Might be worthwhile just fixing up that blocking query support, either on the end of here, or the next iterative PR you do on this.

Actually, that grafana link is the other thing that isn't functional as is - maybe we should catch up on that and get that in here (or straight after) also?

File: ui-v2/app/components/topology-metrics/index.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    // The top/bottom points have the same X position
+    const x = src.x - 3;
+    const topY = src.y + src.height * 0.25 - 5;
+    const bottomY = src.y + src.height * 0.25 + 5;
+
+    const middleX = src.x + 7;
+    const middleY = src.y + src.height / 4;
+
+    return `${x} ${topY} ${middleX} ${middleY} ${x} ${bottomY}`;
+  }
```
[the remark this answers is not in the document]
Comment (kaxcode) [reply ui-v2/app/components/topology-metrics/index.js]: I've implemented the markers! 🎉 THANK YOU

**Comment by kaxcode** [comment]
@johncowen I have the `@metricsHref` changes locally. I'll push them after we get #8798 merged.

**Review by johncowen — APPROVED** [review]
:shipit:

**Pull request merged**